packages/gatsby/content/advanced/error-codes.md

```diff
@@ -184,7 +184,7 @@ This situation usually happens after you've modified the zip archives from your
 
 A file from the cache has been detected unused by `yarn cache clean`.
```

Comment:
`yarn cache clean` doesn't detect unused files anymore, it happens in the fetch step which is executed by any command that runs an install.

```suggestion
A file from the cache has been detected unused when installing dependencies.
```